src/EditorFeatures/Test2/Workspaces/SymbolDescriptionServiceTests.vb

```diff
@@ -523,7 +522,7 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.Workspaces
         </Document>
     </Project>
 </Workspace>
-            Await TestBasicAsync(workspace, "")
+            Await TestBasicAsync(workspace, "Class System.Object")
```

Comment:
For C#, `var x = null` would show nothing but in VB this is automatically `object` if no type is specified.

Reply:
you might want to consider testing a couple of VB options.  Things like 'option explicit' and 'option strict'.